api/postgres/repository.go

```diff
@@ -9,7 +9,7 @@ import (
 
 //RepositoryItem is the documentation for a repository item declaration
 type RepositoryItem interface {
-	GetAllItems(ctx context.Context) ([]*ent.Item, error)
+	GetAllItems(ctx context.Context, limit, offset *int) ([]*ent.Item, error)
 	CreateItem(ctx context.Context, item model.ItemsInput) (*ent.Item, error)
 	UpdateItem(ctx context.Context, id int, input model.ItemsUpdateInput) (*ent.Item, error)
 	DeleteItem(ctx context.Context, id int) error
```

Comment:
If this not generated prefer to avoid cluttered function names. No need for redundant items on function. Need to refactor on ItemRepository implementation
```suggestion
	GetAll(ctx context.Context, limit, offset *int) ([]*ent.Item, error)
	Create(ctx context.Context, item model.ItemsInput) (*ent.Item, error)
	Update(ctx context.Context, id int, input model.ItemsUpdateInput) (*ent.Item, error)
	Delete(ctx context.Context, id int) error
```

Reply:
Oke, i'll refactoring it